packages/delegation-controller/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@metamask/delegation-controller",
+  "version": "0.1.0",
```

Comment:
We typically don't set the initial version of the package to the version we want to publish. The automated publishing workflow uses bumping to know what to release. So if we pre-set this version then there won't be anything to bump when it comes time to make a new release.

Can you set this to 0.0.0 instead?

``` suggestion
  "version": "0.0.0",